R/run_stress_test.R

```diff
@@ -84,8 +80,7 @@ run_stress_test <- function(asset_type,
     baseline_scenario = baseline_scenario,
     shock_scenario = shock_scenario,
     scenario_geography = scenario_geography,
-    lgd_senior_claims = lgd_senior_claims,
```

Comment:
after merging master, the equivalent adjustments are needed for `run_lrisk()`

Reply:
done